cmd/gaia/init/genesis_accts.go

```diff
@@ -116,7 +116,7 @@ func addGenesisAccount(
 			EndTime:         vestingEnd,
 		}
 
-		if bvacc.OriginalVesting.IsAllGT(acc.Coins) {
+		if bvacc.OriginalVesting.IsAnyGT(acc.Coins) {
```

Comment:
Although it's functionally ok, why not using `IsAllLTE()` instead of reintroducing `IsAnyGT()`?

Reply:
++

Reply:
Sure.